test/integration/acceptance/get_transactions_test.cpp

```diff
@@ -66,12 +66,14 @@ TEST_F(GetTransactions, HaveNoGetPerms) {
   };
 
   auto dummy_tx = dummyTx();
-  IntegrationTestFramework(2)
+  IntegrationTestFramework(1)
```

Comment:
Why is that fixes in this pr?

Reply:
because speed up causes tests to fail more often